Add Prettier

aparlato opened the pull request:
Adds `prettier`, `prettier-plugin-svelte`, `husky`, and `pretty-quick` along with a precommit hook to run prettier on changed files. Also formats all files once.

🧹 ✨

ebrelsford reviewed (changes requested):
I added a commit to make `.husky/pre-commit` executable.

Can we also add a note in the `README` that you need to run `husky install` to install the hooks? At least that's my understanding.

aparlato commented:
> I added a commit to make .husky/pre-commit executable.

Ah, great, thanks! 

> Can we also add a note in the README that you need to run husky install to install the hooks? At least that's my understanding.

@ebrelsford I went ahead and added the `husky install` to the prepare script like we do in `maperture` instead so it'll just run automatically. This was an oversight on my part here.

ebrelsford reviewed (approved):
Perfect, thanks!
